tests/genesis.js

```diff
@@ -1,14 +0,0 @@
-const genesisData = require('ethereumjs-testing').tests.basicTests.genesishashestest
```

Comment:
I assume this test case was removed from the official suite?

Reply:
No, this is just some reorganization, move to ``tests/block.js``.